CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 Change Log
 ==========
 
-Version 1.0.7 *(2019-10-01)*
+Version 1.0.9 *(2019-11-07)*
+----------------------------
+* Session Attributes - allow you to add session-specific information to a new message. [See docs for more information](https://developer.helpscout.com/beacon-2/android/#session-attributes)
+
+Bug Fixes: 
+
+* Removed Stetho dependency [#64](https://github.com/helpscout/beacon-android-sdk-sample/issues/64)
```

Comment:
If we could just close these line items with periods that would be sweet, @scottyab. Dig these 😎